docs/index.rst

```diff
@@ -51,13 +51,15 @@ Download `Bebo for iOS <https://itunes.apple.com/us/app/bebo-private-server-for-
    database_api
    notification_api
    camera_api
+   audio_api
    misc_api
 
 .. toctree::
    :maxdepth: 2
-   :caption: Libraries
+   :caption: Modules
```

Comment:
Changed back to "Libraries" and added "Add-ons" section (hopefully more useful leaderboard-type projects will be made in the future?)